cmd/blockchaincmd/deploy.go

```diff
@@ -1145,58 +1154,78 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 		}
 		ux.Logger.PrintToUser("")
 		if err := messengercmd.CallDeploy([]string{}, deployICMFlags, network); err != nil {
-			return err
-		}
-		if network.Kind != models.Local && !useLocalMachine {
-			if flag := cmd.Flags().Lookup(skipRelayerFlagName); flag != nil && !flag.Changed {
-				ux.Logger.PrintToUser("")
-				yes, err := app.Prompt.CaptureYesNo("Do you want to use set up a local interchain relayer?")
-				if err != nil {
-					return err
+			icmErr = err
+			ux.Logger.RedXToUser("Interchain Messaging is not deployed due to: %v", icmErr)
+		} else {
+			ux.Logger.GreenCheckmarkToUser("ICM is successfully deployed")
+			if network.Kind != models.Local && !useLocalMachine {
+				if flag := cmd.Flags().Lookup(skipRelayerFlagName); flag != nil && !flag.Changed {
+					ux.Logger.PrintToUser("")
+					ux.Logger.PrintToUser("As Interchain Messaging was also deployed into your L1, you may")
```

Comment:
maybe better wording:
Do you want to setup local relayer for the messages to be interchanged, as Interchain Messafing was deployed to your L1

also are we sure if we can use `L1` name and not `subnet`. maybe `blockchain` is a better term there.